test/shopify-cli/theme/dev_server/app_extension_proxy_test.rb

```diff
@@ -223,8 +224,10 @@ def test_pass_pending_templates_to_storefront
             .with(
               body: {
                 "_method" => "GET",
-                "replace_templates" => {
-                  "layout/theme.liquid" => @theme["layout/theme.liquid"].read,
+                "replace_extension_templates" => {
+                  "blocks" => {
+                    "blocks/block.liquid" => @theme[blocks/block.liquid"].read
+                  } 
```

Comment:
im trying to modify AppExtensionProxyTest to remove skips now that we have a properly formatted param builder, but I'm having trouble understanding the theme fixture and where the files are created for tests. i scheduled pairing time to go over this 😄